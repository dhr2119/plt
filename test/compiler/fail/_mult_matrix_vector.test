int main() {
  matrix int[2,2] m;
  vector int[3] v;
  m = [1,2;3,4];
  v = | 1 | 2 | 3 |;
  v = m * v;
  return 0;
}
