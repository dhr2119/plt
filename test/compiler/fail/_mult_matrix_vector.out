Fatal error: exception Exceptions.MismatchedMatricesForMult("Matrix M1(i,j) and vector V(n) must have j = n and be of same type to be multiplied")
